docs/extensibility/snippets/MailDevResource/MailKit.Client/MailKitClientSettings.cs

```diff
@@ -63,35 +63,34 @@ configuration section.
                     """);
         }
 
-        if (Uri.TryCreate(connectionString, UriKind.Absolute, out var uri) is false)
+        var builder = new DbConnectionStringBuilder
+        {
+            ConnectionString = connectionString
+        };
+        
+        if (builder.TryGetValue("Endpoint", out var endpoint) is false)
```

Comment:
We typically still support just a URL, and if the whole connection string isn't a URL, then use a DbConnectionStringBuilder.

See https://github.com/dotnet/aspire/blob/681f2e754dc849f96abafcd2829783e69155abf1/src/Components/Aspire.Qdrant.Client/QdrantClientSettings.cs#L26-L48